src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -332,7 +349,14 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
                         if (ProcessLauncher.Launcher.HasChildProc && ProcessLauncher.Launcher.ChildProc.WaitForExit(5000))
                         {
                             ret = ProcessLauncher.Launcher.ChildProc.ExitCode;
-                            Console.WriteLine($"Process exited with code '{ret}'.");
+                            if(ret != 0)
+                            {
+                                Console.Error.WriteLine($"Process exited with code '{ret}'.");
```

Comment:
This change is what is causing the test failures, the test that is failing is looking for "Process exited with code blah" in the standard out, but now it's writing to standard error. We either need to update the test to look at both or change this back to only write to standard out.